controllers/user.js

```diff
@@ -64,7 +64,12 @@ async function verify(req, res) {
 }
 
 
-function getUnsubscribe(req, res) {
+async function getUnsubscribe(req, res) {
+  const subscriber = await DB.getSubscriberByToken(req.query.token);
+  if (!subscriber) {
+    throw new Error("Verification token not found");
```

Comment:
I mentioned this below too, but almost seems more "in-context" up here...

Should this be an "Invalid token" error, and we should have a separate error/`throw` for the token being "missing"?

Or maybe I'm getting confused with "found" wording. As a user, I supplied a token, but maybe it had a typo or something (missed that last character/digit). But I guess I see the other definition of the token was not found in the database.

Reply:
In these case I mean "not found *in our database*" ... but we can update the message. 

@mheubush - what do you think the error message should say? See https://github.com/mozilla/blurts-server/issues/364 and https://github.com/mozilla/blurts-server/issues/353 for the user flows that can lead to this error.